### DISCUSSION TITLE
Verstandnisfrage  Modbus Proxy / EVCC / Solaredge-modbus-multi

### DISCUSSION THREAD

**Discussion by hno2fe** [WillCodeForCats/solaredge-modbus-multi#627] [General]
In Diversen  Foren/ Youtube Beitrage zu Solar Edge Modus  findet mann die Aussage dass immer ein Mobus Proxy  zu verwenden  ist 
Beispiel:  [https://github.com/Akulatraxas/ha-modbusproxy](https://github.com/Akulatraxas/ha-modbusproxy
Aktuell habe ich das Problem dass meine Akku Ladewerte (In&Out kWh) immer nur nach dem Neustart  bis zum nächsten Tag laufen  und ich schon am überlegen bin dass ich um kurz nach Tageswechsel einen Neustart des Systemes  zu machen , aber anderes Thema ..

Mir geht es jetzt darum wenn ich folgendes am Laufen haben sollte 
- Solaredge-modbus-multi 
- EVCC  mit einem  GO-E Charcer 
benötige ich dann einen solchen Modbus Proxy ? 

Wer kan mir hier Tipps geben ?
Viele Grüße aus dem Stuttgarter Raum 
Olli

**Comment by WillCodeForCats** [comment]
I don't use a proxy with my integration, so I don't think a modbus proxy is required. I've written written in a way that it can work with other programs that open connections without a proxy, but only if they are closing the connection after polling the inverter.

The only time you need a proxy is if you have multiple programs connected to the inverter and at least one of them opens a permanent connection since the inverter can only accept one at a time over modbus/tcp.

**Comment by hno2fe** [comment]
Vielen Dank  für deine Antwort . 

Wie ist es denn wen ich in Home Assitant   die beiden  HACS  Sachen   EVCC   und Solar-Edge-Multi parallel laufen lasse ? 

Vertrag sich dieses ?

**Reply by Uhae** [reply]
I've been using EVCC and solaredge-modbus-multi parallel for quite a while now and everything works well. I'm using the above mentioned modbusproxy addon because the solaredge inverter only accepts one connection. 

Originally I had the SE integration directly connecting to the inverter. Then I installed EVCC and couldn't get EVCC to connect to the inverter as the connection was "blocked" by the SE integration. Now I'm using the above mentioned modbusproxy addon and everything works just fine.

**Reply by hno2fe** [reply]
Vielen Dank für deine Antwort , hast du dann im  Solaredge-Modbus-Multi die IP-Adresse des Modbusproxy Addons  eingegeben ?

**Comment by Uhae** [comment]
Ja, genau. In der Config vom Proxy die IP vom SE Wechselrichter als Upstreamhost eingetragen. Von meinem iPhone gesendetAm 30.07.2024 um 16:52 schrieb hno2fe ***@***.***>:﻿
Vielen Dank für deine Antwort , hast du dann im  Solaredge-Modbus-Multi die IP-Adresse des Modbusproxy Addons  eingegeben ?
